elasticai/creator/nn/quantized_grads/fixed_point/test_quantize_to_fixed_point.py

```diff
@@ -0,0 +1,45 @@
+import torch
+
+from elasticai.creator.nn.quantized_grads.fixed_point import FixedPointConfigV2
+from elasticai.creator.nn.quantized_grads.fixed_point._quantize_to_fixed_point import round_to_fixed_point, clamp, \
+    quantize
+
+def test_round_determinstic_to_fixed_point():
+    conf = FixedPointConfigV2(total_bits=8,
+                           frac_bits=2,
+                           stochastic_rounding=False)
+    x = torch.range(-2, 2, step=0.1, dtype=torch.float32)
+
+    actual = round_to_fixed_point(x, conf.frac_bits)
+    expected = torch.Tensor([-2.0000, -2.0000, -1.7500, -1.7500, -1.5000, -1.5000, -1.5000, -1.2500,
```

Comment:
What's the reason for listing four digits after decimal point?

Reply:
No reason. Changed it back.